demos/lexicon.txt

```diff
@@ -370,6 +396,7 @@ pkcs
 pki
 pkparse
 pkwrite
+plaformimagestate
```

Comment:
spelling: __platformimagestage__

Reply:
it is platform image state, we have valid, invalid, pending commit states I believe.

Reply:
right, but the __t__ is missing from "platform",

Reply:
This is still missing a "t"

Currently it is "plaformimagestate"
Should be "platformimagestate"